src/lib/accessibility/Behaviors/Chat/ChatBehavior.ts

```diff
@@ -0,0 +1,31 @@
+import { Accessibility, FocusZoneMode } from '../../interfaces'
+import * as keyboardKey from 'keyboard-key'
+import { FocusZoneDirection } from '../../FocusZone'
+
+const CHAT_FOCUSZONE_ATTRIBUTE = 'chat-focuszone'
+
+const ChatBehavior: Accessibility = {
+  attributes: {
+    root: {
+      role: 'presentation',
+    },
+  },
+  focusZone: {
+    mode: FocusZoneMode.Wrap,
+    props: {
+      shouldEnterInnerZone: event => keyboardKey.getCode(event) === keyboardKey.ArrowRight,
+      direction: FocusZoneDirection.vertical,
+      defaultActiveElement: `[${CHAT_FOCUSZONE_ATTRIBUTE}] > * > *:last-child`, // select last chat message by default
+      [CHAT_FOCUSZONE_ATTRIBUTE]: '', // allows querying the default active element
+    },
+  },
+  keyActions: {
+    root: {
+      focus: {
```

Comment:
Is this ever used as a `focus` action? Or is it more something like `regainFocus`?

Reply:
It is used when you want to unfocus a message and focus the whole list again, so Juraj and I think that `focus` is an appropriate action name as the only thing it does is it literally focuses the chat. :)